catboost/private/libs/algo/greedy_tensor_search.cpp

```diff
@@ -1063,6 +1075,29 @@ static void MarkFeaturesAsUsed(
     split.IterateOverUsedFeatures(estimatedFeaturesContext, markFeatureAsUsed);
 }
 
+static void MarkFeaturesAsUsedPerObject(
```

Comment:
very similar to MarkFeaturesAsUsedPerObject from line 1101 -- pls use TIndexedSubset<ui32> and avoid copy-paste

Reply:
deleted